### ISSUE TITLE
Warrior Fury Talent Bugged: Booming Voice

### ISSUE THREAD

**Issue by Jiokko** [dalaranwow/dalaran-wow#4386]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The talent Called "Booming Voice" at the Fury Warrior talent tree when is specced 2/2 is not giving the expected increased duration of Commanding Shout nor Battle Shout. If you have this talent specced 1/2 (while you have your glyphs) you 'll get a 5mins duration. When you specc the talent 2/2 you still have a 5 mins duration of these 2 buffs.
**Expected behaviour**: Tell us what should happen instead
When you have Glyph of Command/Glyph of Battle and you have this talent specced 2/2 you supposed to get a 6 minutes buff (of either commanding or battle shout).
**Steps to reproduce the problem**:
1. Get an 80 lvl warrior with Glyph of Command + Glyph of Battle.
2. Spec 1 point at Booming Voice. Check the duration.
3. Spec the 2nd point at Booming Voice. Check the duration.

**Include proofs for this behaviour**
i can't upload a photo right now, but i will if needed.
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
this works fine on PTR.

http://i.imgur.com/qaoZph2.jpg
http://i.imgur.com/sk5ciWf.jpg
http://i.imgur.com/cwTJ2Qe.jpg
http://i.imgur.com/6cGvnFx.jpg

**Comment by Jiokko** [comment]
can you open it again? i wanna post the screenshots.

**Comment by Jiokko** [comment]
says 6 minute at the buff text: https://postimg.org/image/5hh2ebpu5/
just buffed, duration, 5 minutes.. :https://postimg.org/image/rf0ckughj/
glyphed and specced 2/2 at booming voice: https://postimg.org/image/h8jfw3rtv/

the only thing i can think of is if: its different if you first put the glyph on and then spec 2/2 or first specc 2/2 and the put the glyph on, ill try reputting the glyph on.

**Comment by Rushor** [comment]
yes i know, all works fine on PTR. the fix for this will be merged to the live server in the future.
